### ISSUE TITLE
Investigate why some users can't access eFolder Express using IE 9 on VA Network

### ISSUE THREAD

**Issue by KHarshawat** [department-of-veterans-affairs/caseflow-efolder#173]
One user at OGC tried to get to eFolder express from his desktop workstation by trying the following:

- efolder.cf.ds.va.gov
- https://efolder.cf.ds.va.gov
- http://efolder.cf.ds.va.gov

 Instead he got to the Bing search results for efolder.cf.ds.va.gov.

**Comment by orischwartz** [comment]
I can't reproduce this so we'll need to reach out to the user directly for more information (see production screenshot pasted in Slack)

**Comment by KHarshawat** [comment]
@lakohl would you mind trying to help us test this out with a user at OGC? We'd like to figure out why this user can't get access to eFolder express from their workstation.  We could reach out to John Matthews who was experiencing this problem I believe.

**Comment by KHarshawat** [comment]
Actually there's already an email thread going that John, Shane, Ori, Giuseppe and I are on titled `EFolder Express`. Last update was `Tuesday, August 16, 2016 at 4:33 PM`. @orischwartz would you mind responding to the thread and setting up some time with John to debug. You can use Join.me, Lync, or GoTomeeting.

**Comment by orischwartz** [comment]
Follow up sent, will call if don't hear back next week. BTW, 99% of the time an issue like this is user error (mistyped URL). Also, John reported this in late July and hasn't reported anything since. But I'll follow up.

**Comment by orischwartz** [comment]
User is on IE 9 and no security settings can be changed without administrator access. It's likely an over-aggressive firewall or and IE9 `trusted sites` setting.

Next steps: 
* Asked user to follow up with helpdesk to see if they could unblock. 
* Will research more to see if it's something that can be fixed without admin access
* I also suggest we move this thread to a private repo because it is unrelated to the application code in this repository.

**Comment by orischwartz** [comment]
Closing in favor of https://github.com/department-of-veterans-affairs/appeals-pm/issues/654 in a private repo
